Source/Diagnostics/BTDiagnostics.H

```diff
@@ -169,11 +169,11 @@ private:
     /** max grid size used to generate BoxArray to define output MultiFabs */
     int m_max_box_size = 256;
 
-    /** Vector of lab-frame time corresponding to each snapshot */
-    amrex::Vector<amrex::Real> m_t_lab;
-    /** Vector of user-defined physical region for diagnostics in lab-frame
-     *  for each back-transformed snapshot */
-    amrex::Vector<amrex::RealBox> m_snapshot_domain_lab;
+    ///** Vector of lab-frame time corresponding to each snapshot */
+    //amrex::Vector<amrex::Real> m_t_lab;
+    ///** Vector of user-defined physical region for diagnostics in lab-frame
+    // *  for each back-transformed snapshot */
+    //amrex::Vector<amrex::RealBox> m_snapshot_domain_lab;
```

Comment:
These lines are commented out and can probably be removed.

Reply:
I put these lines back and made them virtual functions in the main diag class